examples/config-openid.json

```diff
@@ -0,0 +1,78 @@
+{
+  "distSpecVersion": "1.1.0-dev",
+  "storage": {
+    "rootDirectory": "/tmp/zot",
+    "dedupe": true
+  },
+  "http": {
+    "address": "127.0.0.1",
+    "port": "8080",
+    "realm": "zot",
+    "auth": {
+      "htpasswd": {
+        "path": "test/data/htpasswd"
+      },
+      "openid": {
+        "providers": {
+          "github": {
+            "clientid": "client_id",
+            "clientsecret": "client_secret",
+            "keypath": "",
+            "scopes": ["read:org", "user", "repo"]
+          },
+          "google": {
+            "issuer": "https://accounts.google.com",
+            "clientid": "client_id",
+            "clientsecret": "client_secret",
+            "scopes": ["openid", "email"]
+          },
+          "gitlab": {
+            "issuer": "https://gitlab.com",
+            "clientid": "client_id",
+            "clientsecret": "client_secret",
+            "scopes": ["openid", "read_api", "read_user", "profile", "email"]
+          },
+          "dex": {
+            "issuer": "http://127.0.0.1:5556/dex",
+            "clientid": "client_id",
+            "clientsecret": "client_secret",
+            "scopes": ["openid", "user", "email", "groups"]
+          }
+        },
+        "secureCookieEncryptionKey": "test1234test1234",
+        "secureCookieHashKey": "test1234test1234"
+      },
+      "sessionCookieStorePath": "/tmp/cookiestore",
```

Comment:
Should not be exposed, should be determined by the root dir if storage is local. In case of cloud, use in-memory alternative.